castai/resource_eks_cluster.go

```diff
@@ -57,6 +57,12 @@ func resourceEKSCluster() *schema.Resource {
 				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotWhiteSpace),
 				Description:      "AWS region where the cluster is placed",
 			},
+			FieldClusterToken: {
```

Comment:
Can we introduce this field without braking change and then do removal of `cluster_token` resource? This way users will be able to use this field without migrating to major version